src/code/z_scene.c

```diff
@@ -536,57 +536,58 @@ void Scene_SetExitFade(PlayState* play) {
     play->transitionType = Entrance_GetTransitionFlags(play->nextEntrance) & 0x7F;
 }
 
+static void (*sSceneCmdHandlers[])(PlayState*, SceneCmd*) = {
```

Comment:
remove static ? (idk how you handle static in MM, I've also seen it removed from overlay symbols)

Reply:
I don't think it's set and stone once data is named, but I think the tendency is to not have it if it's a unique enough name, but it's not been a huge deal. This is unique enough to remove though, so I'll do it